tests/test_page.py

```diff
@@ -60,14 +60,14 @@ def test_filter_entries(har_data):
     entries = page.filter_entries(request_type='.*ET')
     assert len(entries) == 4
     for entry in entries:
-        assert entry['request']['method'] == 'GET'
+        assert entry.request.method == entry["request"]["method"] == 'GET'
```

Comment:
I think I already mentioned this but maybe we should leave the existing tests that operate on the dicts intact? This would give extra coverage to the backwards compatibility which is super important, and you have separate tests that operate directly on the `HarEntry`.

Reply:
I changed it to `assert entry.request.method == entry["request"]["method"] == 'GET'` as that way it will check that the dict method equals the the new method.
If you would like I can revert the changes and just create a new test for the new method.

Reply:
Hahaha I did not even catch that, nice work, this is a great idea!